sdk/appconfiguration/azure-data-appconfiguration/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
```

Comment:
This version change will be done by updating version_client.txt an version update script. Like we had discussion , I will create a PR for this change now.

Reply:
Thanks. Will revert this version changes